Base/res/html/misc/border-radius.html

```diff
@@ -0,0 +1,230 @@
+            border: 1px solid black;
+            border-radius: 20px;
+        }
+
+        .box-1-6 {
+            border: 1px solid black;
+            border-top-width: 1px;
+            border-right-width: 5px;
+            border-bottom-width: 10px;
+            border-left-width: 12px;
+            border-radius: 20px;
+        }
+
+        .box-2 {
+            border: 1px solid black;
+            border-top-left-radius: 10px;
+        }
+
+        .box-1-7 {
+            border: 5px solid black;
+            border-radius: 15px;
+        }
+
+        .box-1-8 {
+            border: 20px solid black;
+            border-radius: 10px;
+        }
+
+        .box-1-9 {
+            border: 20px inset black;
+            border-radius: 10px;
+        }
+
+        .box-1-10 {
+            border: 20px outset black;
+            border-radius: 10px;
+        }
+
+        .box-3 {
+            border: 1px solid black;
+            border-top-right-radius: 10px;
+        }
+
+        .box-4 {
+            border: 1px solid black;
+            border-bottom-right-radius: 10px;
+        }
+
+        .box-5 {
+            border: 1px solid black;
+            border-bottom-left-radius: 10px;
+        }
+
+        .box-6 {
+            width: 200px;
+            height: 200px;
+            border: 1px solid black;
+            border-radius: 75px;
+        }
+
+        .box-7 {
+            background-color: magenta;
+            border-radius: 10px;
+        }
+
+        .box-7-1 {
+            width: 200px;
+            height: 200px;
+            background-color: magenta;
+            border-radius: 25px 50px 75px 100px;
+        }
+
+        .box-7-2 {
+            width: 200px;
+            height: 200px;
+            background-color: magenta;
+            border-radius: 100px 75px 50px 25px;
+        }
+
+        .box-8 {
+            background-color: magenta;
+            border-radius: 20px;
+        }
+
+        .box-9 {
+            background-color: magenta;
+            border-top-left-radius: 10px;
+        }
+
+        .box-10 {
+            background-color: magenta;
+            border-radius: 15px;
+        }
+
+        .box-11 {
+            background-color: magenta;
+            border-top-right-radius: 10px;
+        }
+
+        .box-12 {
+            background-color: magenta;
+            border-bottom-right-radius: 10px;
+        }
+
+        .box-13 {
+            background-color: magenta;
+            border-bottom-left-radius: 10px;
+        }
+
+        .box-14 {
+            background-color: lime;
+            border: 1px solid black;
+            border-radius: 500px;
+        }
+
+        .box-15 {
+            background-color: red;
+            border: 1px solid black;
+            border-radius: 500px;
+        }
+    </style>
+</head>
+
+<body>
+    <h3>Border-radius</h3>
+    <p>The boxes are 50x50px and have a background-color</p>
+    <em>All round 10px</em>
+    <div class="box box-7"></div>
+    <br />
```

Comment:
Given that this is using a HTML5 doctype and `<br>` is a void element in that case, can we remove the slash across the board?

Reply:
I'm so used to adding the / to self closing tags...
It's done.